server/datastore/sql/migrations/001-initial.sql

```diff
@@ -15,3 +15,21 @@ CREATE TABLE posts (
   postedAt INTEGER NOT NULL,
   FOREIGN KEY (userId) REFERENCES users (id)
 );
+
+CREATE Table comments (
+  id      VARCHAR PRIMARY KEY,
+  userId  VARCHAR NOT NULL,
+  PostId  VARCHAR NOT NULL,
```

Comment:
let's keep camelCase throughout our column names for consistency, and to reduce the chance of bugs when writing our SQL queries.

Reply:
fixed.